index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+
+<body>
+    <section class="head">
+      <header>
+      <div class="background-image-1"></div>
+      <ul class="status-bar">
+        <li class="time-text">1:00</li>
+        <li class="group-1">
+          <img class="inside" src="icons/Cellular-Connection.svg" alt="cellular" id="connection">
+          <img class="inside" src="icons/Wifi.svg" alt="Wifi" id="wifi">
+          <img class="inside" src="icons/Battery.svg" alt="battery" id="battery">
+        </li>
+      </ul>
```

Comment:
**Optional Comment**
I must commend your effort to make this project look similar to the Figma design as much as possible :handshake: 
However, I think this part of the header is not necessary. The time, network, battery, and wireless connection icons are there to emulate a mobile view. These icons are not embedded in the webpage, it's a feature provided by the mobile device operating system. Therefore, I strongly believe that you should remove it :blush: